org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/splitting/AntlrLexerSplitter.java

```diff
@@ -213,6 +222,41 @@ private static String getVarnameFromDecl(String varDecl) {
 			return null;
 		return m.group(1);
 	}
+	
+	void copyTailAndRefactorDfaTransitionS() {
```

Comment:
package private by intention. if so why not private as the caller?

Reply:
good catch

Reply:
I copied it from `boolean copyUntilMethod()` ... will make both private